internal/cmd/identities.go

```diff
@@ -36,10 +36,6 @@ func newIdentitiesCmd() *cobra.Command {
 	return cmd
 }
 
-type addIdentityCmdOptions struct {
-	Password bool `mapstructure:"password"`
-}
```

Comment:
This wasn't being used, so I removed it. Should it be used? I don't see any flags on `identity add`, so I'm guessing they were removed.